aip/0203.md

```diff
@@ -3,40 +3,45 @@ aip:
   id: 203
   state: approved
   created: 2018-07-17
+  updated: 2019-06-17
 permalink: /203
 redirect_from:
   - /0203
 ---
 
 # Field behavior documentation
 
-When defining fields in protocol buffers, it is customary to describe that
-field's behavior (e.g., whether it is required or optional) at the beginning of
-the field's comments. For example:
+When defining fields in protocol buffers, it is customary to explain to users
+certain aspects of the field's behavior (e.g., whether it is required or
+optional).
 
-```proto
-// Required. The audio data to be recognized.
-RecognitionAudio audio = 2;
-```
+Additionally, it can be useful for other tools to understand this behavior (for
```

Comment:
Put this into the same paragraph, rather than having two single-sentence paragraphs?

Reply:
Done.